config/crd/about.k8s.io_clusterproperties.yaml

```diff
@@ -25,9 +26,7 @@ spec:
     name: v1alpha1
     schema:
       openAPIV3Schema:
-        description: ClusterProperty is a resource provides a way to store identification
-          related, cluster scoped information for multi-cluster tools while creating
-          flexibility for implementations.
+        description: ClusterProperty is the Schema for the clusterproperties API
```

Comment:
I believe we want to retain the previous more-descriptive text. Looking at the repo, it looks like there's actually been 2 copies of the API created - one tree living in the top level dir and one tree under clusterproperty/

I believe we now want to use the top level, but they are not quite in sync. Do you mind making the updates to pkg/apis/v1alpha1 instead?

Reply:
Can you please provide a bit more clarification? I mean I have updated the CRDs in both the top dir and manifests residing in cluster property. Do you want me to change it only for the top level or something else.

Reply:
Ah! Got it. They both had some minor differences. I missed out on that. Will regenerate the top dir crd with pkg/v1alpha1 annotations. Also, do you want me to retain the updated customize in clusterproperty.